tests/test_utils.py

```diff
@@ -66,17 +66,41 @@ def test_convert_to_float():
 
 def test_parse_json_with_comments():
 
-    # Need to add comments
-    json_with_comments = ("""{"key1": "value1", /*some comments */\n"""
-                          """/*more comments */\n"""
-                          """"key2": "value2", "key3": 5}""")
+    json_with_comments = """
+    {
+        "key1": "value1", /*some comments */
+        /*more comments */
+        "key2": "value2", 
+        "key3": 5
+    }"""
 
     tempf = tempfile.NamedTemporaryFile(mode='w', suffix='.csv', delete=False)
```

Comment:
I wonder why we chose to use "csv" as the extension?

Reply:
Should I change it to .json?

Reply:
Yeah, I think it would be best.

Reply:
Agreed.